website/docs/usage/large-language-models.mdx

```diff
@@ -340,15 +340,45 @@ A _task_ defines an NLP problem or question, that will be sent to the LLM via a
 prompt. Further, the task defines how to parse the LLM's responses back into
 structured information. All tasks are registered in the `llm_tasks` registry.
 
-Practically speaking, a task should adhere to the `Protocol` `LLMTask` defined
-in [`ty.py`](https://github.com/explosion/spacy-llm/blob/main/spacy_llm/ty.py).
-It needs to define a `generate_prompts` function and a `parse_responses`
-function.
+Practically speaking, a task should adhere to the `Protocol` named `LLMTask`
+defined in
+[`ty.py`](https://github.com/explosion/spacy-llm/blob/main/spacy_llm/ty.py). It
+needs to define a `generate_prompts` function and a `parse_responses` function.
 
-| Task                                                                        | Description                                                                                                                                                  |
-| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| [`task.generate_prompts`](/api/large-language-models#task-generate-prompts) | Takes a collection of documents, and returns a collection of "prompts", which can be of type `Any`.                                                          |
-| [`task.parse_responses`](/api/large-language-models#task-parse-responses)   | Takes a collection of LLM responses and the original documents, parses the responses into structured information, and sets the annotations on the documents. |
+Tasks may support prompt sharding (for more info see the API docs on
+[sharding](/api/large-language-models#task-sharding) and
+[non-sharding](/api/large-language-models#task-nonsharding) tasks). The function
+signatures for `generate_prompts` and `parse_responses` depend on whether they
+do.
+
+| _For tasks *not supporting* sharding:_ | Task | Description |     |
+| -------------------------------------- | ---- | ----------- | --- |
```

Comment:
Looks like this table format needs to be fixed (cf https://deploy-preview-13214--spacy.netlify.app/usage/large-language-models#tasks)

Reply:
Fixed. Dunno how that happened.